wandb/sdk/artifacts/artifact.py

```diff
@@ -1572,26 +1457,11 @@ def get(self, name: str) -> Optional[data_types.WBValue]:
         Arguments:
             name: The artifact relative name to get
 
+        Returns:
+            An `ArtifactManifestEntry` object.
```

Comment:
```suggestion
            A `W&B` object.
```

Reply:
sorry about this and the one above. i accidentally viewed the methods above these and said ArtifactManifestEntry object.

Reply:
I'm a bit confused by this suggestion to replace ArtifactManifestEntry -> W&B object. What is a W&B object? Is there an actual object that is of class W&B?

Reply:
I'm just maintaining consistency between this and the other methods that take a w&b object as input. For example, see `artifact.set()`. The input is data_types.WBValue. I guess instead of W&B Object, we could write that this returns `data_types.WBValue `instead?

Reply:
Without getting into the weeds of how Python attributes work under the hood (PyTypeObjects)... let's describe what [W&BValue](https://github.com/wandb/wandb/blob/d10f170b4cfc9280a0e4eb7284fcfe1a0addb6c9/wandb/sdk/data_types/base_types/wb_value.py#L55) is (with an emphasis on how customers should use the returned object)?

Reply:
I think we should change it to W&B Value - 

Btw I don't think we need to put extra info in this return type this should belong in class not artifacts class.

Reply:
True. But we don't have WBValue surfaced anywhere....I think that would require additional conversations.

Reply:
Ok gotcha I hadn't realized users cannot see this class. The definition in the class is:
`Typed objects that can be logged with `wandb.log()` and visualized by wandb.

    The objects will be serialized as JSON and always have a _type attribute that
    indicates how to interpret the other fields.`
so I think we can Returns: `a W&B object which can then be be logged with `wandb.log()` and visualized in the W&B UI.`